.storybook/preview.js

```diff
@@ -1,3 +1,16 @@
+import { withPerformance } from "storybook-addon-performance";
+import { Theme } from "../src/Theme";
+import { CSSReset } from "@chakra-ui/react";
+
+export const decorators = [
+  withPerformance,
+  (Story) => (
+    <Theme>
+      <CSSReset />
```

Comment:
We don't currently use the reset in the app right? We probably should.

Reply:
Good callout - I'll bring that in too